includes/util.py

```
@@ -0,0 +1,110 @@
+    with open(sourcefile, 'rb') as src:
+        while os.path.getsize(destfile) < dstsize:
+            data = src.read(BUF_SIZE)
+            if not data:
+                src.seek(0)
+                continue
+            dst.write(data)
+    dst.close()
+
+def parse_timefile(filen):
+    ''' Parse output from perf or time '''
+    if cfgConfig['use_perf']:
```

Ruff (0.14.13)

Undefined name `cfgConfig`

(F821)